app/pb_kits/playbook/pb_circle_chart/docs/_circle_chart_colors.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { CircleChart } from '../../'
+
+const dataWithColors = [
+  {
+    name: 'Waiting for Calls',
+    value: 41,
+  },
+  {
+    name: 'On Call',
+    value: 49,
+  },
+  {
+    name: 'After call',
+    value: 10,
+  },
+]
+
+const CircleChartColors = () => (
+  <div>
+    <CircleChart
+        chartData={dataWithColors}
+        colors={['data-6', 'data-4', 'data-2']}
```

Comment:
I wonder if there is a more direct way to provide these props. It doesn't seem like devs will immediately make the connection to tokens.

Reply:
Yeah Jason and I threw out a bunch of ideas when we were building the extension of custom colors, and this was the syntax we landed on. I agree though. Could definitely be revisited. Fwiw, I didn't implement that logic on this PR, so i'm guessing it would need to be a separate ticket?